snpdb/models/models_variant.py

```diff
@@ -242,32 +242,109 @@ class AlleleMergeLog(TimeStampedModel):
     message = models.TextField(null=True)
 
 
-@dataclass(frozen=True)
+@dataclass
 class VariantCoordinate(FormerTuple):
+    """ This stores coordinates, when you want to use it, be sure to call either:
+            * as_external_explicit() - if you want to interface with eg VCF
+            * as_internal_symbolic() - to store in internal Database
+    """
     chrom: str
-    pos: int
+    start: int
+    end: int
     ref: str
     alt: str
 
+    def __init__(self, chrom, start, end: int = None, ref: str = None, alt: str = None):
```

Comment:
To make the typehint happy it should be

end: Optional[int] = None, ref: Optional[str] = None, alt: Optional[str] = None

as None is not a valid int or str